jupyter-extension/src/plugins/mount/__tests__/mount.test.tsx

```diff
@@ -127,37 +133,50 @@ describe('mount plugin', () => {
   });
 
   it('should generate the correct layout', async () => {
-    const plugin = new MountPlugin(app, docManager, factory, restorer, tracker);
+    const plugin = new MountPlugin(
+      app,
+      docManager,
+      factory,
+      restorer,
+      widgetTracker,
+    );
     expect(plugin.layout.title.caption).toBe('Pachyderm Mount');
     expect(plugin.layout.id).toBe('pachyderm-mount');
     expect(plugin.layout.orientation).toBe('vertical');
-    expect(plugin.layout.widgets).toHaveLength(8);
+    expect(plugin.layout.widgets).toHaveLength(9);
     expect(plugin.layout.widgets[0]).toBeInstanceOf(ReactWidget);
     expect(plugin.layout.widgets[1]).toBeInstanceOf(ReactWidget);
     expect(plugin.layout.widgets[2]).toBeInstanceOf(ReactWidget);
     expect(plugin.layout.widgets[3]).toBeInstanceOf(ReactWidget);
-    expect(plugin.layout.widgets[4]).toBeInstanceOf(FileBrowser);
-    expect(plugin.layout.widgets[5]).toBeInstanceOf(ReactWidget);
+    expect(plugin.layout.widgets[4]).toBeInstanceOf(ReactWidget);
+    expect(plugin.layout.widgets[5]).toBeInstanceOf(FileBrowser);
     expect(plugin.layout.widgets[6]).toBeInstanceOf(ReactWidget);
     expect(plugin.layout.widgets[7]).toBeInstanceOf(ReactWidget);
+    expect(plugin.layout.widgets[8]).toBeInstanceOf(ReactWidget);
   });
 
   it('return from pipeline view to the correct layout', async () => {
-    const plugin = new MountPlugin(app, docManager, factory, restorer, tracker);
-    const pipeline = plugin.layout.widgets[3];
-    const fileBrowser = plugin.layout.widgets[4];
+    const plugin = new MountPlugin(
+      app,
+      docManager,
+      factory,
+      restorer,
+      widgetTracker,
+    );
+    const pipelineSplash = plugin.layout.widgets[3];
+    const fileBrowser = plugin.layout.widgets[5];
     expect(fileBrowser).toBeInstanceOf(FileBrowser);
 
     plugin.setShowConfig(false);
-    expect(pipeline.isHidden).toBe(true);
+    expect(pipelineSplash.isHidden).toBe(true);
     expect(fileBrowser.isHidden).toBe(false);
 
     plugin.setShowPipeline(true);
-    expect(pipeline.isHidden).toBe(false);
+    expect(pipelineSplash.isHidden).toBe(false);
     expect(fileBrowser.isHidden).toBe(true);
 
     plugin.setShowPipeline(false);
-    expect(pipeline.isHidden).toBe(true);
+    expect(pipelineSplash.isHidden).toBe(true);
     expect(fileBrowser.isHidden).toBe(false);
   });
 });
```

Comment:
It would be good to have a test which tests showing the splash screen (if it's not too much work to set up the pre-conditions)